Ethnic/ETH1219baryas.xml

```diff
@@ -30,23 +30,36 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+         <langUsage>
+            <language ident="gez">Gǝʿǝz</language>
+            <language ident="en">English</language>
+         </langUsage>
+         <abstract>
+            <ab>Idiom or a name previously used for <persName ref="ETH1946Nara"/>, later for a wider group of 'blacks', a social group, 
+               similar to <persName ref="ETH2043sanqel"/>, later perceived as pejorative - as reflected by the 
+               <ref type="authFile" corresp="Barya">name for the demonic entity</ref>. Also name of a military unit in premodern Ethiopia.</ab>
+         </abstract>
       </profileDesc>
       <revisionDesc>
          <change who="PL" when="2016-10-18"> Created file from google
                                 spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: ethnic</change>
+         <change when="2024-09-17" who="CH">Updated name and added bibliography.</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <personGrp role="ethnic">
-               <persName xml:id="n1">
-                  <orgName>barya, soc. group</orgName>
-               </persName>
+               <persName xml:id="n1" xml:lang="gez"><orgName>ባርያ፡</orgName></persName>
```

Comment:
(I am actually not sure whether `gez `or `am`, but we can leave gez as the texts are mainly gez)